examples/gopher 4.nwk

```diff
@@ -0,0 +1,2 @@
+((h6, h7) h1, (h8, (h10, (h12, (h14, (h16, h17) h15) h13) h11) h9) h2) h0;
```

Comment:
Just reset this file to the one in master -- https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

Reply:
Removed.